xinference/model/llm/core.py

```diff
@@ -244,7 +244,7 @@ def create_llm_model_instance(
             )
     else:
         model = llm_cls(
-            model_uid, llm_family, llm_spec, quantization, save_path, kwargs
+            model_uid, llm_family, llm_spec, quantization, save_path, **kwargs
```

Comment:
@ChengjieLi28 Changed the way `kwargs` pass, please see if there are any side effects

Reply:
> @ChengjieLi28 Changed the way `kwargs` pass, please see if there are any side effects

Cannot change here. Otherwise previous parameters would not work.

Reply:
fix.